### ISSUE TITLE
Agenda, August 24, 2023

### ISSUE THREAD

**Issue by chrishtr** [web-platform-tests/interop#397]
* Commit to September 14 as the launch date of the 2024 proposal period
* State of HTML overview
* Review and land the [charter PR](https://github.com/web-platform-tests/interop/pull/102)
* Review and unblock landing of the [PR to explain the Interop project further](https://github.com/web-platform-tests/interop/pull/394)
* Review and finalize the [2024 focus areas issue template](https://github.com/web-platform-tests/interop/pull/385)
* Review and take action on [test change proposals](https://github.com/web-platform-tests/interop/issues?q=is%3Aissue+is%3Aopen+label%3Atest-change-proposal)
* Discuss [voting procedures](https://github.com/web-platform-tests/interop/pull/390) for the 2024 focus areas

**Comment by nairnandu** [comment]
Attendees: @chrishtr, @LeaVerou, @meyerweb, @nt1m, @nairnandu, @gsnedders, @slightlyoff, @boazsender, @dandclark, @rnyman

*   Commit to September 14 as the launch date of the 2024 proposal period
    *   Pending Mozilla's position (no attendance from Mozilla today), consensus to move forward with Sept 14th launch.
*   State of HTML overview - leaverou
    *   Expected to launch mid-September.
    *   Survey design is [here](https://coda.io/@leaverou/state-of-html/survey-outline-7). Team can also provide feedback in the discussion/feedback section [here](https://github.com/Devographics/surveys/discussions)
    *   What features would be useful to ask about (awareness, usage, sentiment)? 
    *   boazsender: this data can be used as a prioritization rubric for Interop 2024
    *   leaverou: yes, thats absolutely the idea. Here is the [roadmap](https://github.com/Devographics/surveys/discussions/112) 
    *   leaverou: this also includes HTML-adjacent APIs like PWAs, Web Components etc.
    *   [ ] Next step: leaverou will file an issue in Interop repo to solicit feedback from the group. 
        * Issue created #398 

*   Review and land the [charter PR](https://github.com/web-platform-tests/interop/pull/102)
    *   nairnandu: any new edits needed? How do we progress?
    *   gsnedders: We would want some text about what decisions are not public. This is something that needs to be added as a suggestion
    *   slightlyoff: we continue to have a strong interest making sure governance is open and transparent
    *   gsnedders: our preference would be closed-by-default.
    *   slightlyoff: Is there some way we can be more specific about the types of decisions we make public vs private? 
    *   [ ] Next step: dandclark will draft a suggested edit to the charter for the team to review. gsnedders will review the draft and provide feedback.

*   Review and unblock landing of the [PR to explain the Interop project further](https://github.com/web-platform-tests/interop/pull/394)
    *   Some pending comments that need to be resolved
    *   chrishtr: looks good.
    *   boazsender: skimmed through it and looks good for the most part. Adding comments
    *   dandclark: looks good
    *   meyerweb: no objections to landing it once the comments are resolved
    *   chrishtr: propose that we land it once Sam has a chance to respond to all open comments
*   Review and finalize the [2024 focus areas issue template](https://github.com/web-platform-tests/interop/pull/385)
    *   Consensus on moving forward with this template. Land the PR on September 14th
*   Review and take action on [test change proposals](https://github.com/web-platform-tests/interop/issues?q=is%3Aissue+is%3Aopen+label%3Atest-change-proposal)
    *   #375 - what is the next step?
        *   Next step: nt1m will post a list of tests that need to be excluded
    *   #380 - it seems like the spec is unclear. Need more inputs on this one
*   Discuss [voting procedures](https://github.com/web-platform-tests/interop/pull/390) for the 2024 focus areas
    *   dandclark: the public nature of decision making as proposed for the charter should be considered here as well
    *   meyerweb: seems comprehensive
    *   boazsender: all for having decision making be done in public
    *   dandclark: for the proposals we eliminate, how would we communicate to the authors
    *   chrishtr: same as last year?
    *   boazsender: talking about would be a good idea. Maybe how we communicate to authors could be a different PR
    *   chrishtr: agree with making that a separate PR
    *   chrishtr: the #n selection is a concern. Suggest that each org put forward a reasonable # of proposals, without setting an arbitrary number.

**Comment by jgraham** [comment]
> Pending Mozilla's position (no attendance from Mozilla today), consensus to move forward with Sept 14th launch.

We are happy with the Sept. 14th proposal.
